proposals/feast-adoption.md

```diff
@@ -0,0 +1,91 @@
+
+![][diagrams/aiml_lifecycle.png]
+
+2\. It will also allow the Data WG to have an answer for the online serving of features. Additionally, this will nicely complement the Spark Operator as [Feast supports batch and stream processing using Spark as an offline store](https://docs.feast.dev/reference/offline-stores/spark).
+
+3\. The Feast community is healthy and the users will further grow the Kubeflow community.
+
+4\. Feast is expanding its scope to support Generative AI and RAG as a first-class citizen (retrieval/vector search in particular), which will help ensure Kubeflow has a solution for RAG.
+
+5\. With the inclusion of Feast, we can provide end-to-end demos of development and production AI/ML and we can also provide suggested patterns for stitching the Kubeflow products together so that MLOps engineers, ML Engineers, and AI engineers can be impactful immediately after deploying Kubeflow.
+
+6\. We are just as committed to Feast as we have ever been and we believe this will meaningfully enhance Kubeflow and result in Kubeflow getting the benefit of my contributions and the contributions of the Feast community.
+
+Beyond the benefits to Kubeflow, I believe it will provide significant benefits to Feast in growing our community and embedding ourselves in a complete platform. This will also allow both communities (particularly Data Scientists/MLEs) to benefit from the GenAI/LLM/NLP work that I have been doing (see here: [\#4964](https://github.com/feast-dev/feast/issues/4964) and [\#4364](https://github.com/feast-dev/feast/issues/4364)).
+
+As clear in the [Kubeflow User Survey](https://15905978862850875460.googlegroups.com/attach/150ca032927e6/'22%20Kubeflow%201.6%20User%20Survey%20-%20initial%20results.pdf?part=0.1.1&view=1&view=1&vt=ANaJVrFmCCfq8rEbnzakW7JKVPbTl1nhLDckzg79h1xmup5qc_c3rftVy1ne7tkCUMf5pUOfS3aMEXHfKEKeEVwV1TtdQOnvx3XMHYZ2uQHv-70K5CRyV0M), Feast serves a critical and challenging need: feature engineering and feature serving.
+
+![][diagrams/kubeflow_feast.png]
+![][diagrams/kubeflow_feature_engineering.png]
+
+#### **Kubeflow Working Group Data**
+
+As mentioned, Feast will also allow the [Data WG](https://github.com/kubeflow/community/pull/673) to have an answer for the online serving of features and will nicely complement the Spark Operator as [Feast supports batch and stream processing using Spark as an offline store](https://docs.feast.dev/reference/offline-stores/spark).
+
+The scope of this WG is to provide guidelines for AI/ML users to run data preparation tools such as [Spark](https://spark.apache.org/), [Dask](https://www.dask.org/), [Ray](https://docs.ray.io/en/latest/) on Kubernetes, [Model Registry](https://github.com/kubeflow/kubeflow/issues/7396), and PIpelines can be a part of the **Data WG** scope.
+
+## **Maintainers**
+
+The following maintainers are committed to maintain Spark Operator under Kubeflow GitHub org to track issues, do releases, merge PRs.
```

Comment:
"The following maintainers are committed to maintain Spark Operator under Kubeflow GitHub org to track issues, do releases, merge PRs." should it not be about feast?

Reply:
Thanks for catching that! Fixed!